### PR TITLE
Add play from playlist

### LINKED ISSUE #6: Add Play feature in Playlist
The Playlist only has the option to `delete` a song from Playlist. Add a feature to `play` any song from the Playlist.

### PR DESCRIPTION
**Pull request by zee-bit**
Closes #6 and #3

### CODE REVIEW

**Review by abhisheknaiidu — COMMENTED** [review]
Looks Great to me, but should we remove `Whole Playlist Part`?
Let me know what you think, I think it's better to remove that part.

**Comment by zee-bit** [comment]
I don't see any reason to remove the `Playlist` part bhaiya. Personally, I think it looks great and is also an important feature that every Song Player should probably have. Again, if you have something else planned in mind in place of `Playlist` and that's why you want to remove this part then it's fine with me. I can handle it !!

File: src/components/SongPlayer.js [hunk 1]
```diff
@@ -91,12 +95,36 @@ function SongPlayer() {
...
+        // }
+        // else if( played >= 0.99 && state.isRepeating ) {
+        //     setPlayed(0)
+        //     if(!firstSong) {
+        //         reactPlayerRef.current.seekTo(played);
+        //     }
+        //     else {
+        //         dispatch( {type: 'SET_SONG', payload: { song: firstSong }})
+        //     }
+        // }
```
Comment (abhisheknaiidu) [inline src/components/SongPlayer.js]: @zee-bit I mean this part, it's better to remove, I guess

Comment (zee-bit) [reply src/components/SongPlayer.js]: Ohh...Yes..yes. I'll remove that part. LOL

**Review by abhisheknaiidu — APPROVED** [review]

**Pull request merged**